src/test/modelmanager_test.cpp

```diff
@@ -112,17 +112,20 @@ class MockModelManager : public ovms::ModelManager {
     }
 };
 
-TEST(ModelManager, ConfigParseNoModels) {
+class ModelManager : public ::testing::Test {
```

Comment:
Doesn't it shadow ovms::ModelManager? Even if not, isnt it misleading?

Reply:
It's in different namespace